console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -2559,8 +2553,10 @@ const resolvers = {
       runOperationForAll(args.input, t => deleteIotProject(t));
       return true;
     },
-    disableIotProjects: (parent, args) => {
-      runOperationForAll(args.input, t => disableIotProject(t));
+    toggleIoTProjectsStatus: (parents, args) => {
+      runOperationForAll(args.input, t =>
+        toggleIoTProjectStatus(t, args.state)
```

Comment:
Can you try to keep consistency for args. Use status instead state.
args.status

Reply:
Done